apple/internal/apple_framework_import.bzl

```diff
@@ -223,6 +256,25 @@ def _apple_static_framework_import_impl(ctx):
     if ctx.attr.weak_sdk_frameworks:
         objc_provider_fields["weak_sdk_framework"] = depset(ctx.attr.weak_sdk_frameworks)
 
+    imported_swiftmodules = [
+        f
+        for f in header_imports
+        if _is_swiftmodule(f.basename)
+    ]
+
+    if imported_swiftmodules:
+        swift_toolchain = ctx.attr._toolchain[SwiftToolchainInfo]
+        swift_linkopts = swift_common.swift_runtime_linkopts(True, swift_toolchain)
+        if ctx.var["COMPILATION_MODE"] == "dbg":
```

Comment:
rules_swift does this for fastbuild too https://github.com/keith/rules_swift/blob/2c84d8615ca7046c343d747545b7ec77fe4a7c71/swift/internal/api.bzl#L315-L335

Reply:
@allevato same here, could we reuse some logic from rules_swift here to avoid duplicating the logic to check when to add these flags?

Reply:
Ditto vvv